napari/utils/_tests/test_notebook_display.py

```diff
@@ -0,0 +1,18 @@
+import numpy as np
+
+from napari.utils.notebook_display import nbscreenshot
```

Comment:
Do we want to expose `nbscreenshot` at the `napari.utils` level so one could just type
`from napari.utils import nbscreenshot`?

I think that would be nice